src/routes/Routing.tsx

```diff
@@ -0,0 +1,14 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Dashboard from "../components/Dashboard";
+import Login from "../components/Login";
+import Profile from "../components/Profile";
+
+export default function Routing() {
+  return (
+    <Routes>
+      <Route path="/profile" element={<Profile />} />
+      <Route path="/" element={<Dashboard />} />
+      <Route path="/login" element={<Login />} />
```

Comment:
We will need to have a fall back page, 
so this will be "/anything" navigate to page not found may be